core/templates/dev/head/collection_editor/settings_tab/CollectionDetailsEditorDirective.js

```diff
@@ -43,13 +43,21 @@ oppia.directive('collectionDetailsEditor', [function() {
             };
           }
         );
+        $scope.getLanguageListForSelect = function() {
```

Comment:
Is there a reason why this is a function but $scope.TAG_REGEX is just a constant?